### PR TITLE
Backend init

### PR DESCRIPTION
**Pull request by iamAbasu7**
converting base64 data to buffer then running the network

### CODE REVIEW

File: packages/backend/src/middleware/upload.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import multer from 'multer';
```
Comment (radhe-zeeve) [inline packages/backend/src/middleware/upload.ts:1]: Remove `multer` as it is not used in the application

File: packages/backend/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "@types/http-errors": "^2.0.1",
+    "@types/multer": "^1.4.7",
+    "@types/node": "^18.14.0",
+    "@typescript-eslint/eslint-plugin": "^5.53.0",
+    "eslint": "^8.34.0",
+    "eslint-config-standard-with-typescript": "^34.0.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^15.6.1",
+    "eslint-plugin-promise": "^6.1.1",
+    "multer": "^1.4.5-lts.1",
```
Comment (radhe-zeeve) [inline packages/backend/package.json:33]: Remove multer as it is not used in the application

**Review by radhe-zeeve — CHANGES_REQUESTED** [review]

File: packages/backend/package.json (hunk 2 above)
Comment (iamAbasu7) [reply packages/backend/package.json:33]: Yeah!

**Review by radhe-zeeve — APPROVED** [review]
Approved

**Pull request merged**